peregrine/run.py

```diff
@@ -302,7 +307,8 @@ def main():
                                stage2_loop_filter_params=stage2_params,
                                tracker_options=tracker_options,
                                output_file=args.file,
-                               progress_bar_output=args.progress_bar)
+                               progress_bar_output=args.progress_bar,
+                               check_l2c_mask=args.check_l2c_mask)
```

Comment:
Let's also add the same parameter for Tracker in peregrine/peregrine/analysis/tracking_loop.py. It is a dedicated script for tracking performance analysis.